src/components/implementation/no_interface/scheddev/sl.h

```diff
@@ -174,10 +192,23 @@ sl_cs_exit(void)
 	if (!ps_cas(&sl__globals()->lock.u.v, cached.v, 0))    goto retry;
 }
 
+static inline microsec_t
```

Comment:
Just a note that the math in these is expensive, and prone to wrap-around issues.  We don't want to use it much.  This code is perfectly good, this is just a note to you for the future.

Concretely:  we want to keep things in cycles, in cycles where possible.